cmd/tiller-proxy/internal/handler/apprepos_handler.go

```diff
@@ -143,23 +173,98 @@ func (a *appRepositoriesHandler) Create(w http.ResponseWriter, req *http.Request
 		return
 	}
 
+	secret := secretForRequest(appRepoRequest, *appRepo)
```

Comment:
nit: why using the content of `appRepo` instead of the pointer?

Reply:
No reason, updated.